### PR TITLE
Feature/sdl 0205 custom button subscription

### PR DESCRIPTION
**Pull request by KhrystynaDubovyk**
❗  NOTE  ❗  : This update is based on [Button Subscription Response from HMI](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/253), so this PR should be merged AFTER #253  

Guidelines updates according to the proposal [SDL-0205 Avoid Custom button subscription in case HMI incompatibility](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0205-Avoid_custom_button_subscription_when_HMI_does_not_support.md)

This PR is **ready** for review.

### Summary
Update SubscribeButton index and assets files, Buttons/GetCapabilities index

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: docs/Buttons/SubscribeButton/index.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+## SubscribeButton
+
+Type
+: Function  
+
+Sender
+: SDL
+
+Purpose
+: Subscribe mob app to a particular button
```
Comment (ashwink11) [inline docs/Buttons/SubscribeButton/index.md:10]: @KhrystynaDubovyk Could you please correct the typo "mob app"?

File: docs/Buttons/UnsubscribeButton/index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+## UnsubscribeButton
+
+Type
+: Function  
+
+Sender
+: SDL
+
+Purpose
+: Unsubscribe mob app from a button
```
Comment (ashwink11) [inline docs/Buttons/UnsubscribeButton/index.md:10]: @KhrystynaDubovyk Could you please correct the typo "mob app"?

**Review by ashwink11 — CHANGES_REQUESTED** [review]

**Comment by ashwink11** [comment]
Hi @KhrystynaDubovyk, the file docs/Buttons/UnsubscribeButton/index.md in this PR and in PR #253 seems to be the same. Do we need this file in both PR's?

**Comment by ashwink11** [comment]
@jordynmackool this PR is ready for review and is approved by Ford.

File: docs/Buttons/GetCapabilities/index.md [hunk 3]
```diff
@@ -10,13 +10,20 @@ Purpose
...
 
 #### Parameters
 
 This RPC has no additional parameter requirements
 
 ### Response
 
+!!! must  
+
+Check hmi_capabilities, and respond to SDL providing the capabilities of supported buttons.
```
Comment (iCollin) [inline docs/Buttons/GetCapabilities/index.md:23]: Could we please add a note here that if the HMI wishes to support soft buttons it must include CUSTOM_BUTTON in these capabilities?

**Comment by iCollin** [comment]
Could you please merge in develop?

File: docs/Buttons/SubscribeButton/assets/SubscribeCUSTOM_BUTTON.txt [hunk 4]
```diff
@@ -0,0 +1,18 @@
+participant App
```
Comment (iCollin) [inline docs/Buttons/SubscribeButton/assets/SubscribeCUSTOM_BUTTON.txt:1]: @KhrystynaDubovyk Let's add this diagram to `docs/Buttons/SubscribeButton/index.md`

Comment (iCollin) [reply docs/Buttons/SubscribeButton/assets/SubscribeCUSTOM_BUTTON.txt:1]:
```
diff --git a/docs/Buttons/SubscribeButton/index.md b/docs/Buttons/SubscribeButton/index.md
index 7c78176..537909e 100644
--- a/docs/Buttons/SubscribeButton/index.md
+++ b/docs/Buttons/SubscribeButton/index.md
@@ -46,6 +46,11 @@ SubscribeButton
 ![SubscribeButton](./assets/SubscribeButton.png)
 |||
 
+|||
+SubscribeButton CUSTOM_BUTTON
+![SubscribeButton CUSTOM_BUTTON](./assets/SubscribeCUSTOM_BUTTON.png)
+|||
+
 ### JSON Message Examples
 
 #### Example Request
```

**Review by iCollin — APPROVED** [review]

**Pull request merged**